public/vis_controller.ts

```diff
@@ -101,6 +102,9 @@ export function getEnhancedTableVisualizationController(
         } else {
           updateScope();
         }
+      } catch (error) {
+        reject(error);
+        }
```

Comment:
could you indent nicely try/catch block and inside lines?

Reply:
I committed a new version with improved indentation, would you mind to review it?